src/nvim/buffer.c

```diff
@@ -3565,26 +3559,29 @@ void get_rel_pos(win_T *wp, char_u *buf, int buflen)
         : (int)(above * 100L / (above + below)));
 }
 
-/*
- * Append (file 2 of 8) to "buf[buflen]", if editing more than one file.
- * Return TRUE if it was appended.
- */
-static int 
-append_arg_number (
-    win_T *wp,
-    char_u *buf,
-    int buflen,
-    int add_file                   /* Add "file" before the arg number */
+/// Append (file 2 of 8) to "buf[buflen]", if editing more than one file.
+/// Return true if it was appended.
+static bool append_arg_number(
+  win_T *wp,
+  char_u *buf,
+  int buflen,
+  int add_file                   /* Add "file" before the arg number */
 )
```

Comment:
Fun fact: I looked at the style guide and they are actually supposed to be 4-spaced.